content/4_Gold/Divis.mdx

```diff
@@ -502,7 +493,7 @@ $$
 
 ```cpp
 ll gcd(ll a, ll b){ return b == 0 ? a : gcd(b,a%b); }
-ll lcm(ll a, ll b) { return a/gcd(a,b)*b; }
+ll lcm(ll a, ll b)1 { return a/gcd(a,b)*b; }
```

Comment:
?

Reply:
Yes, that shouldn't be there. I don't know what happened with it.

Reply:
```suggestion
ll lcm(ll a, ll b) { return a/gcd(a,b)*b; }
```